src/hooks/useProjectGalaxyCampaignsInfo.ts

```diff
@@ -50,19 +50,13 @@ const useProjectGalaxyCampaignsInfo = () => {
     variables: { account, campaignId: 'GCD5yUt73A'},
     ...querySetting
   })
-
-  const campaignInfo_GCTANUUJkf = useQuery(PROJECT_GALAXY_CAMPAIGN_INFO, {
-    variables: { account, campaignId: 'GCTANUUJkf'},
-    ...querySetting
-  })
-
-  const campaignInfo_GCuq6UU5zS = useQuery(PROJECT_GALAXY_CAMPAIGN_INFO, {
-    variables: { account, campaignId: 'GCuq6UU5zS' },
+  const campaignInfo_GCxTdUwieL = useQuery(PROJECT_GALAXY_CAMPAIGN_INFO, {
+    variables: { account, campaignId: 'GCxTdUwieL'},
     ...querySetting
   })
 
-  const campaignInfo_GCSH6Utrps = useQuery(PROJECT_GALAXY_CAMPAIGN_INFO, {
-    variables: { account, campaignId: 'GCSH6Utrps'},
+  const campaignInfo_GCv7dUwr2V = useQuery(PROJECT_GALAXY_CAMPAIGN_INFO, {
```

Comment:
I miss to review this before... i think u could use only single query to get all data